extensions/wikia/CommunityPage/models/CommunityPageSpecialUsersModel.class.php

```diff
@@ -127,12 +127,12 @@ function () {
 					->AND_( 'rev_timestamp' )->GREATER_THAN( $dateTwoYearsAgo )
 					->AND_( 'wup_property' )->EQUAL_TO( 'editcount' )
```

Comment:
If we removed `wup_value` I think we can also remove `wup_property` and whole `LEFT JOIN`